coremltools/converters/keras/_keras_converter.py

```diff
@@ -11,8 +11,8 @@
 from ...models import datatypes
 from ...models import MLModel as _MLModel
 from ...models import _MLMODEL_FULL_PRECISION, _MLMODEL_HALF_PRECISION, _VALID_MLMODEL_PRECISION_TYPES
-from ...models.utils import convert_neural_network_weights_to_fp16 as convert_neural_network_weights_to_fp16
-from ...models.utils import convert_neural_network_spec_weights_to_fp16 as convert_neural_network_spec_weights_to_fp16
+from ...models.utils import _convert_neural_network_weights_to_fp16 as _convert_neural_network_weights_to_fp16
```

Comment:
Yeah, same as my previous comment. I guess the code was redundant before.

Reply:
updated